doc/classes/Object.xml

```diff
@@ -8,7 +8,7 @@
 		You can construct Objects from scripting languages, using [code]Object.new()[/code] in GDScript, or [code]new Object[/code] in C#.
 		Objects do not manage memory. If a class inherits from Object, you will have to delete instances of it manually. To do so, call the [method free] method from your script or delete the instance from C++.
 		Some classes that extend Object add memory management. This is the case of [RefCounted], which counts references and deletes itself automatically when no longer referenced. [Node], another fundamental type, deletes all its children when freed from memory.
-		Objects export properties, which are mainly useful for storage and editing, but not really so much in programming. Properties are exported in [method _get_property_list] and handled in [method _get] and [method _set]. However, scripting languages and C++ have simpler means to export them.
+		Object export properties are mainly useful for storage and editing, but not really so much in programming. Properties are exported in [method _get_property_list] and handled in [method _get] and [method _set]. However, scripting languages and C# have simpler means to export them.
```

Comment:
"export" is used as a verb here, so I don't see why this wasn't grammatically?

"script languages and C#" is weird because C# one of said scripting languages.

Reply:
Also, C++ exposes properties with the `ADD_PROPERTY` macro, which is likely what this sentence refers to.

Reply:
Not sure what of make of that whole sentence, myself. #67880 completely gets rid of the C++ mention because the majority of users do not even need this information.

Reply:
"can export" may be better to break up 3 words that look like nouns. (well, "can" can also be a noun, but rather unexpected in this context)

Reply:
Ah, somehow my brain never parsed that "export" could be a verb here. "can" would clarify, though I guess it was technically correct to begin with.